docs/meshcloud.metadata-tags.md

```diff
@@ -27,6 +27,11 @@ Administrators can configure tags so that only [Partner users](./administration.
 
 The [administration documentation](./administration.index.md) provides further details on how partner users can edit restricted tags.
 
+## Immutable tags
+
+Administrators can configure immutable tags. This means that we allow the selection of specific tag values only during the creation process [meshCustomers](meshcloud.customer.md), [meshProjects](meshcloud.project.md), [meshLandingZones](meshcloud.landing-zones.md) and [meshPaymentMethods](./meshcloud.payment-methods.md). Afterwards it is no longer possible to edit the selected values. Also administrators cannot change immutable tags after creation.
```

Comment:
d: so our plan is once a tag is immutable we can unset  the immutable in tag catalogue right?
or what does it mean after creation?

Reply:
Just check the implementation, you are able to unset it.
```suggestion
Administrators can configure immutable tags. This means that we allow the selection of specific tag values only during the creation process [meshCustomers](meshcloud.customer.md), [meshProjects](meshcloud.project.md), [meshLandingZones](meshcloud.landing-zones.md) and [meshPaymentMethods](./meshcloud.payment-methods.md). Afterwards it is no longer possible to edit the selected values. Also administrators cannot change immutable tag values after creation.
```

Reply:
You can remove the immutable flag from the tags. This works fine.